cdc/sink/dmlsink/mq/worker.go

```diff
@@ -310,9 +328,45 @@ func (w *worker) sendMessages(ctx context.Context) error {
 	}
 }
 
+func (w *worker) claimCheckSendMessage(ctx context.Context, topic string, partition int32, message *common.Message) error {
+	if w.claimCheck == nil {
```

Comment:
this process makes me very confused . what is the purpose of NewClaimCheckMessage?

Reply:
`NewClaimCheckMessage` create the message which contains the `claim check location`, it should be send to the kafka after the original message is delivered to the external storage.